### ISSUE TITLE
"send_resolved: true" doesn't respect  "resolve_timeout" value

### ISSUE THREAD

**Issue by fvigotti** [prometheus/alertmanager#551]
using latest version of alertmanager && prometheus with hipchat receiver 

I've created an alert based on time that gets triggered after 1 minute, and last for about 2 minutes,
if I have `send_resolved: false` I correctly receive 2 alerts, but if I set `send_resolved: true`
after the alerts is resolved, I get about 15 to 30 more alerts ( 1 every minute) with a *Status* being *resolved* even playing with *global.resolve_timeout* value change nothing.. even setting this to 2 minutes.. seems to not have effect.. 
also anyway shouldn't only one of those **resolved** message pass ? 

Does this works in your setups ? 

Thanks,
Francesco

**Comment by fabxc** [comment]
Hey,

can you provide some exact steps to reproduce the problem – ideally you'd try to model the Prometheus behavior with pure curl commends like:

```bash
curl -XPOST -d'[{"labels":{"alertname":"a"}}]' http://localhost:9093/api/v1/alerts
```

**Comment by brian-brazil** [comment]
No response from reporter, and this is likely the set of issues we fixed in 0.6.0.
